source/distributed/p4est_wrappers.cc

```diff
@@ -191,8 +191,6 @@ namespace internal
     size_t (&functions<2>::connectivity_memory_used) (types<2>::connectivity *p4est)
       = p4est_connectivity_memory_used;
 
-    const unsigned int functions<2>::max_level = P4EST_MAXLEVEL;
```

Comment:
Even though you put the initializer into the `.h` file, you still need to tell the compiler to allocate a global memory location for this variable (in case, for example, someone wanted to pass around the address of the `static const` variable). So you need to leave this line here, but remove the initializer (which is now at the declaration). Same in line 382.